src/ui/Logo/Logo.ts

```diff
@@ -9,7 +9,11 @@ import { IComponentBindings } from '../Base/ComponentBindings';
 import { ComponentOptions } from '../Base/ComponentOptions';
 import { Initialization } from '../Base/Initialization';
 
-export interface ILogoOptions {}
+export type ValidLogoTargetOptions = `_top` | `_blank` | `_self` | `_parent`;
```

Comment:
Just wondering if the type name is a bit redundant because of the "Valid" preffix. I think the type definition implies that the values are valid. But out of consistency with existing types like ValidLayout, it's ok :)

Oh, and is there a reason for the backticks if there is no string interpolation?

Reply:
For the backticks, no reason, except I copy pasted them from the doc ;) I'll change it to single quotes.